ci-operator/step-registry/api-job/k8sgpt/api-job-k8sgpt-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+openai_token=$(cat "/var/run/vault/tests-private-account/openai-token")
+
+k8sgpt version
+k8sgpt auth add -m gpt-3.5-turbo -b openai -p ${openai_token} | tee ${ARTIFACT_DIR}/result || true
```

Comment:
@jianzhangbjz suggest change `${ARTIFACT_DIR}/result` to `${ARTIFACT_DIR}/k8sgpt-result`